client/__tests__/util/helpers.ts

```diff
@@ -0,0 +1,273 @@
+import { ElementHandle, expect, Locator, Page } from "@playwright/test";
+
+import { ERROR_NO_TEST_ID_OR_LOCATOR } from "../common/constants";
+import { waitUntilNoSkeletonDetected } from "../e2e/cellxgeneActions";
+
+export const TIMEOUT_MS = 3 * 1000;
+export const WAIT_FOR_TIMEOUT_MS = 3 * 1000;
+
+// (seve): We use TEST_ENV to describe the environment that playwright is running against. Sometimes the FE tests are run against a local instance of the app which points at a deployed instance of the backend.
```

Comment:
wait I don't see `TEST_ENV` variable being used anywhere. Maybe I'm missing something here lol

Reply:
accidentally left in from dp copy